.github/workflows/_integration-tests.yaml

```diff
@@ -16,14 +16,16 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 1
+      - name: Get prefsrc
+        run: |
+          echo "IPADDR=$(ip -4 -j route get 2.2.2.2 | jq -r '.[] | .prefsrc')" >> $GITHUB_ENV
       - name: Setup operator environment
         uses: charmed-kubernetes/actions-operator@main
         with:
-          juju-channel: 2.9/stable
+          juju-channel: 3.0/stable
           provider: microk8s
-          bootstrap-options: "--agent-version 2.9.34"
-      - name: Patch hostpath-provisioner
-        run: >
-          sg microk8s -c "kubectl patch deployment hostpath-provisioner -n kube-system -p '{\"spec\": {\"template\": {\"spec\": {\"containers\": [{\"name\":\"hostpath-provisioner\", \"image\": \"cdkbot/hostpath-provisioner:1.3.0\" }] }}}}'"
+          channel: 1.25-strict/stable
+          microk8s-addons: "hostpath-storage dns metallb:${{ env.IPADDR }}-${{ env.IPADDR }}"
+          bootstrap-options: "--agent-version 3.0.2"
```

Comment:
We could probably kill this line also, and use the same agent version as the Juju version in the stable track.

Reply:
Agreed. We should be testing against the latest release